### DISCUSSION TITLE
KILITLER ACILINCA

### DISCUSSION THREAD

**Discussion by mlkmutlu** [IBAX-io/go-ibax#947] [Soru Cevap - Türkçe]
IBXC LER SERBEST KALINCA SIZCE KAC USD OLUR

**Comment by zukenberger** [comment]
Bilmiyoruz ama.Referansı çok olanın daha çok olur gibime geliyor. O yüzden burdan herkes biribirini yazdığına yorum yapsın ve okuna tıklasın .

**Comment by Yusufknk** [comment]
Bunu ilerleyen zaman gösterecek.
